config/environments/test.rb

```diff
@@ -47,6 +45,6 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
-  # Run resque tasks as part of the main app
-  Resque.inline = true
```

Comment:
Do we need this?

Reply:
since the test run without it, i'd say no (`:inline` is the default for `ActiveJob`)